templates/html.html.twig

```diff
@@ -1,19 +1,15 @@
 <html class="no-js">
   <head>
-    <meta charset=utf-8> 
+    <meta charset=utf-8>
     <title>Code Enigma | Json Populate</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0">
     <meta http-equiv="cleartype" content="on">
     <meta name="HandheldFriendly" content="True">
     <meta name="MobileOptimized" content="320">
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <link type="text/css" rel="stylesheet" href="https://code.jquery.com/ui/1.12.0/themes/smoothness/jquery-ui.css" media="all" />
     <link type="text/css" rel="stylesheet" href="/assets/css/style.css" media="all" />
-    <script src="//code.jquery.com/jquery-3.2.1.min.js" type="text/javascript"></script>
-    <script src="//code.jquery.com/jquery-migrate-3.0.0.min.js" type="text/javascript"></script>
-    <script src="//code.jquery.com/ui/1.12.1/jquery-ui.min.js" type="text/javascript"></script>
-    <script src="//unpkg.com/split.js/split.min.js" type="text/javascript"></script>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:400,700" rel="stylesheet">
```

Comment:
I'll change this later to match our new branding. There's a look and feel ticket with me but this is fine for now.